teku/src/main/java/tech/pegasys/teku/cli/slashingprotection/ExportCommand.java

```diff
@@ -69,13 +71,14 @@ public void run() {
     if (!errors.isEmpty()) {
       errors.forEach((key, error) -> SUB_COMMAND_LOG.display("ERROR: " + error));
       SUB_COMMAND_LOG.exit(
-          1, "There were errors reading from slashing protection files, cannot complete.");
+          FATAL_EXIT_CODE,
+          "There were errors reading from slashing protection files, cannot complete.");
     }
     try {
       SUB_COMMAND_LOG.display("Writing slashing protection data to: " + toFileName);
       slashingProtectionExporter.saveToFile(toFileName, SUB_COMMAND_LOG::display);
     } catch (IOException e) {
-      SUB_COMMAND_LOG.exit(1, "Failed to export slashing protection data.", e);
+      SUB_COMMAND_LOG.exit(FATAL_EXIT_CODE, "Failed to export slashing protection data.", e);
```

Comment:
in export command it's not a service, these are probably less important... i'd probably leave them at code 1, but I don't feel super strongly
My logic here is with a CLI tool a 2 literally indicates you've used it wrongly... so like you've given me an option that isn't compatible...